src/tools/perf/Makefile.am

```diff
@@ -46,7 +50,10 @@ libucxperf_la_LIBADD = \
 
 ucx_perftest_SOURCES  = perftest.c
 ucx_perftest_LDFLAGS    = \
-     $(RTE_LDFLAGS)
+     $(RTE_LDFLAGS) \
+     $(OPENMP_CFLAGS)
```

Comment:
Nop, adding OPENMP_CFLAGS to LDFLAGS is the autoconf blessed solution. 
https://www.gnu.org/software/autoconf/manual/autoconf-2.67/html_node/Generic-Compiler-Characteristics.html